go/api/commands.go

```diff
@@ -154,4 +213,104 @@ type StringCommands interface {
+	//
+	// For example:
+	//  result, err := client.Strlen("key");
+	//
+	// [valkey.io]: https://valkey.io/commands/strlen/
+	Strlen(key string) (int64, error)
+
+	// Overwrites part of the string stored at key, starting at the specified offset, for the entire length of value.
+	// If the offset is larger than the current length of the string at key, the string is padded with zero bytes to make
+	// offset fit.
+	// Creates the key if it doesn't exist.
+	//
+	// See [valkey.io] for details.
+	//
+	// Parameters:
+	//  key    - The key of the string to update.
+	//  offset - The position in the string where value should be written.
+	//  value  - The string written with offset.
+	//
+	// Return value:
+	//  The length of the string stored at key after it was modified.
+	//
+	// For example:
+	//  result, err := client.SetRange("key", 6, "GLIDE");
+	//
+	// [valkey.io]: https://valkey.io/commands/setrange/
+	SetRange(key string, offset int, value string) (int64, error)
+
+	// Returns the substring of the string value stored at key, determined by the offsets start and end (both are inclusive).
+	// Negative offsets can be used in order to provide an offset starting from the end of the string. So -1 means the last
+	// character, -2 the penultimate and so forth.
+	//
+	// See [valkey.io] for details.
+	//
+	// Parameters:
+	//  key   - The key of the string.
+	//  start - The starting offset.
+	//  end   - The ending offset.
+	//
+	// Return value:
+	//  A substring extracted from the value stored at key.
+	//
+	// For example:
+	//  result, err := client.GetRange("mykey", -3, -1);
+	//
+	// [valkey.io]: https://valkey.io/commands/getrange/
+	GetRange(key string, start int, end int) (string, error)
+
+	// Appends a value to a key. If key does not exist it is created and set as an empty string, so APPEND will be similar to
+	// SET in this special case.
+	//
+	// See [valkey.io] for details.
+	//
+	// Parameters:
+	//  key   - The key of the string.
+	//  value - The value to append.
+	//
+	// Return value:
+	//  The length of the string after appending the value.
+	//
+	// For example:
+	//  result, err := client.Append("key", "value");
+	//
+	// [valkey.io]: https://valkey.io/commands/append/
+	Append(key string, value string) (int64, error)
+
+	// Returns the longest common subsequence between strings stored at key1 and key2.
+	//
+	// Since:
+	//  Valkey 7.0 and above.
+	//
+	// Note:
+	//  When in cluster mode, key1 and key2 must map to the same hash slot.
+	//
+	// See [valkey.io] for details.
+	//
+	// Parameters:
+	//  key1 - The key that stores the first string.
+	//  key2 - The key that stores the second string.
+	//
+	// Return value:
+	//  A String containing the longest common subsequence between the 2 strings.
+	//  An empty String is returned if the keys do not exist or have no common subsequences.
+	//
+	// For example:
+	//  result, err := client.LCS("testKey1", "testKey2");
```

Comment:
We need to consider showing the result of a command. Could be very helpful for commands with complex output, like `lcsIdx`.

Reply:
Added for all the commands.